pyrdp/layer/tcp.py

```diff
@@ -46,7 +45,7 @@ class TwistedTCPLayer(IntermediateLayer, Protocol):
     TCP observers are notified when a connection is made.
     """
 
-    def __init__(self, config: MITMConfig):
```

Comment:
Pretty sure you can just put quotes around the class name to avoid importing it.